checklists/azure_storage_checklist.en.json

```diff
@@ -456,6 +456,18 @@
       "id": "C01.04",
       "severity": "Medium",
       "link": "https://learn.microsoft.com/azure/storage/blobs/soft-delete-blob-enable?tabs=azure-portal"
+    },
+    {
+      "category": "Security",
```

Comment:
@erjosito I added this section here. I'm not sure what the ID or the guide is. Anywhere I can

Reply:
@cshea-msft 

Per the [JSON schema](https://github.com/Azure/review-checklists/blob/main/checklists/checklist.schema.json#L38):
id = "A locally unique ID (only for this checklist) used primarily for sorting."
guid = "A globally unique GUID for the recommendation."

The id just needs to be unique within this file and should ideally fit within the conventions or naming sequence for other items that have existing id values. The GUID needs to be unique across all JSON files in the repo. You can use something like https://guidgenerator.com/ to get a globally unique value.

Reply:
thanks @sdolgin updated with id and guid